website/content/api-docs/secret/consul.mdx

```diff
@@ -93,14 +95,13 @@ To create a client token within a particular Consul admin partition:
 
 ### Parameters for Consul versions 1.8 and above
 
-- `consul_namespace` `(string: "")` <EnterpriseAlert inline /> - Specifies the Consul namespace the token
-  will be generated within. The namespace must exist, and the policies or roles assigned to the
-  Vault role must also exist inside the given Consul namespace. If not provided, the "default"
-  namespace is used.
+- `consul_namespace` `(string: "")` - Specifies the Consul namespace the token
```

Comment:
Suggestion:
Specifies the Consul admin partition in which the token is generated within.